bzira

```diff
@@ -182,28 +217,12 @@ bz_tojira() {
 		]
 	}
 }
-EOF
-
-	cat <<EOF > "$issue_remotelink_payload"
-{
-	"relationship": "Bugzilla",
-	"object":
-	{
-		"title": "Bug ${bz_id}",
-		"summary": "${bz_summary}",
-		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=${bz_id}",
-		"icon":
-		{
-			"url16x16": "https://bugzilla.redhat.com/extensions/RedHat/web/css/favicons/production.ico"
-		}
-	}
-}
 EOF
 
 	issue_create < "$issue_payload" | tee "$issue_response_payload"
 
-	issue_id="$(jq -r '.key' "$issue_response_payload")"
-	issue_remotelink_create "$issue_id" < "$issue_remotelink_payload" > /dev/null
+	issue_key="$(jq -r '.key' "$issue_response_payload")"
```

Comment:
do you need the outside quotes?

Reply:
In this case, I use quotes mostly out of habit.

1. I could remove the ‘-r’ flag and have ‘jq’ quote the value (it’s expected to be a string);
2. I could trust Jira not to use spaces or weird characters in keys.

That being said, putting quotes myself is a no-brainer.

Unless I am missing something?